app/models/manageiq/providers/embedded_terraform/automation_manager/template.rb

```diff
@@ -1,4 +1,4 @@
-class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::ConfigurationScriptPayload < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptPayload
+class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::Template < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptPayload
```

Comment:
@agrare do we need a migration to update existing objects in the database with the old class name?

Reply:
@jrafanie since we haven't actually included this plugin in the core Gemfile yet the only people impacted will be those of us who have developed this plugin so I don't think we need a data migration.